Standards/scs-0106-v1-evaluate-current-state-of-cloudmon.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Evaluating the deployment of CloudMon in SCS infrastructure
+type: Decision Record
+---
+
+## Introduction
+
+In the fast-paced environment of modern cloud computing, ensuring the reliability and performance
+of cloud infrastructure is paramount for organizations. Effective monitoring and testing framework
+are essential tools in this endeavor, enabling proactive identification and resolution of issues before
+they impact critical operations. However, selecting the right monitoring solution can be a daunting task,
+particularly when faced with options that are overly complex and lack essential documentation and working
+scenarios.
+
+## Motivation
+
+One such solution, the [Cloudmon](https://stackmon.org/) project, faces challenges that might limit its effectiveness for organizations looking
+for efficient and reliable monitoring capabilities. This introduction outlines the reasons why our organization opts
+against utilizing the Cloudmon project and instead embraces a more streamlined and effective approach involving Gherkin
+test scenarios and mapping Python API calls to interact with OpenStack resources. By addressing the complexities and
+shortcomings of the Cloudmon project, SCS organization aims to adopt a monitoring solution that not only meets but
+exceeds our requirements for simplicity, reliability, and ease of use.
```

Comment:
Honestly this phrasing is more then subjective. After dozens of hours spend explaining the documentation, setup scenarios, testing capabilities, workshops this statement is very offensive. SCS organization entered the evaluation and asked for multiple workshops to understand the tooling and offered help in improving the documentation. There was absolutely no output.
What you offer as an alternative is another "one of thousands" of namely universal testing frameworks that were not designed to monitor cloud environment and are not really dealing with measuring low level performance (of the API itself).